people/views.py

```diff
@@ -70,6 +70,23 @@ def profile(request, year, pk):
 		'roles': RecruitmentApplication.objects.filter(user = user, status = 'accepted').order_by('recruitment_period__fair')
 	})
 
+def profile_delete(request, year, pk):
+	fair = get_object_or_404(Fair, year = year)
+	user = request.session['user']
+
+	if request.method == 'POST':
+		Profile.objects.filter(user = request.user).delete()
+		User.objects.filter(pk = pk).delete()
+		return redirect('/accounts/logout?next=/register')
```

Comment:
This redirect seems to be more focused on company user then students as you can login as student from that path. Not sure how we want to do this.

Reply:
Fixed it to regular login for students.